package.json

```diff
@@ -4,6 +4,12 @@
   "description": "Smart contract and DApp by poa.network to create proof of a physical address",
   "main": "server.js",
   "scripts": {
+    "build": "cd web-dapp && npm run build",
+    "deploy": "cd utils && node deploy_contract",
```

Comment:
Could we rename it to `deploy-on-test`?
`deploy` looks too "prod-like", however this script is only for local tests